### DISCUSSION TITLE
Preferences for Taskbar size and font size

### DISCUSSION THREAD

**Discussion by kitiki1222** [lawand-dot-io/taskbar#36] [Feature requests]
Option to adjust the size of Taskbar, making it smaller or larger.

**Comment by lawand-dot-io** [comment]
Hello. Do you mean the height or the width?

**Reply by kitiki1222** [reply]
maybe both if possible

**Comment by semtex1987** [comment]
I'll second this one, on 4k monitor theres plenty of room for taskbar to be taller so that icons and text don't have to be so tiny. If nothing else allow for some configuration by the user to scale the taskbar size a bit up or down.

**Reply by kitiki1222** [reply]
yes

**Reply by lawand-dot-io** [reply]
Interesting points. I never tested Taskbar on a 4k monitor. I will keep these ideas in mind. Thanks for the feedback :)

**Comment by davidgross** [comment]
Another +1 on this, specifically height of the bar.  On my screens the bar is much taller than it needs to be, would be nice to make it shorter.

**Reply by lawand-dot-io** [reply]
Sure. I'll keep this in mind. Out of curiosity which screen size do you mean? I'm guessing something smaller than the average? Maybe a screenshot would give a better perspective. Cheers

**Reply by davidgross** [reply]
I have 22 inch 1080p monitors, and the bar is nearly .5 inches (~1.2cm) tall. I'd like it closer to .25 inches(~0.6cm?).

**Reply by kitiki1222** [reply]
Yes it’s so small so I want to adjust the size

**Reply by davidgross** [reply]
Yes, It needs to be adjustable, because it displays differently on different screens. Also a dpi setting might help, where depending on screen resolution and size, it can adjust itself. On my laptop screen it's ok, but on my monitors, it's way big. I can see on a 4k or larger resolution screen how it would be very small.

**Reply by lawand-dot-io** [reply]
Exactly, I should allow making Taskbar either smaller or larger so that it fits every user's preference, and maybe also make the size change based on dpi or something similar

**Reply by kitiki1222** [reply]
Yes

**Comment by lawand-dot-io** [comment]
Duplicate request closed: https://github.com/lawand-dot-io/taskbar/discussions/45

**Reply by kitiki1222** [reply]
👍🏻

**Comment by adityakurian** [comment]
+1 look forward to this. great app.

**Reply by lawand-dot-io** [reply]
I'm glad you enjoy Taskbar. Size preferences will be implemented at some point before Taskbar requires purchase. Cheers

**Comment by lawand-dot-io** [comment]
Note for reference: consider combining with https://github.com/lawand-dot-io/taskbar/discussions/81

**Comment by lawand-dot-io** [comment]
More context: https://github.com/lawand-dot-io/taskbar/discussions/87

**Comment by lawand-dot-io** [comment]
More context: https://github.com/lawand-dot-io/taskbar/discussions/95

**Comment by lawand-dot-io** [comment]
This task has been changed so that it's only about Taskbar size and font size. Further options are moved to: https://github.com/lawand-dot-io/taskbar/discussions/135

Feel free to upvote that new request to increase the chance it gets implemented. Cheers

**Comment by lawand-dot-io** [comment]
This is now implemented in the latest update of Taskbar (1.3.0) which is now released :)
